priv/static/icon-svg/green-line-b.svg

```diff
@@ -1,8 +1,5 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24px" height="24px" role="img" version="1.1" viewBox="0 0 24 24">
-    <title>
-        green line B
-    </title>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24px" height="24px" role="img" viewBox="0 0 24 24">
```

Comment:
![Screenshot 2024-11-18 at 10 14 06 AM](https://github.com/user-attachments/assets/55511ecc-4935-448a-9c2c-98d722980c31)

Definitely out of scope for this PR, but I would be pretty surprised if we're actually using all of icons in this folder. Maybe at some point it would be worth going through and identifying which icons we still need to keep?

Reply:
Yep. Added to the dev discussion list!